README.md

```diff
@@ -39,7 +39,9 @@ Follow the detailed [installation instructions](docs/Installation.md).
 
 ### Configuration 
 
-Zonemaster *Backend* is configured as a whole from `/etc/zonemaster/backend_config.ini`.
+Zonemaster *Backend* is configured as a whole from `/etc/zonemaster/backend_config.ini`
+(CentOS, Debian and Ubuntu) or `/usr/local/etc/zonemaster/backend_config.ini`
```

Comment:
For CentOS, Debian and Ubuntu it is still /etc/zonemaster in the installation instructions, as it has always been. For FreeBSD there was a mixture of /etc/zonemaster and /usr/local/etc/zonemaster in the instructions, and since the latter is the expected in FreeBSD I decided to make it consistently /usr/local/etc/zonemaster. That affects the installation instructions for FreeBSD and the start files for FreeBSD.

There are no changes to CentOS, Debian or Ubuntu.

Reply:
@sandoche2k, please review again.